angular/lib/src/bootstrap/run.dart

```diff
@@ -46,6 +41,7 @@ Injector _platformInjector() {
 /// Optionally may supply a [createInjector] function in order to provide
 /// services to the root of the application:
 ///
+/// ```
 /// // Assume this file is "main.dart".
 /// import 'package:angular/angular.dart';
 /// import 'main.template.dart' as ng;
```

Comment:
Btw, that's a lot of boilderplate:

```dart
void main() {
  runApp(ng.HelloWorldNgFactory, createInjector: helloInjector);
}

@GenerateInjector(const [
  const ClassProvider(HelloService),
])
final InjectorFactory helloInjector = ng.helloInjector$Injector;
```

Is there any way that this could be simplified to, say:

```dart
@GenerateInjector(const [
  const ClassProvider(HelloService),
])
void main() {
  runApp(ng.HelloWorldNgFactory, createInjector: $Injector);
}
```

Reply:
It could, but then there is more magic.

Reply:
At the very least you'd still need the `ng` prefix on `$Injector`. But then this would rely on that being some magic top level declaration in the generated file, so what happens if you want multiple generated injectors?

I think the current approach makes it more obvious what the annotation does, and provides you the flexibility to generate multiple injectors with whatever name you desire.